core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -1,144 +1,672 @@
+              </div>
+              <div class="row oppia-donate-page-money-main-text">Spreading the word</div>
+              <hr class="w-25 oppia-donate-page-hr">
+              <div class="row oppia-donate-page-money-sub-text">
+                Your donations help with our efforts in raising awareness of Oppia across the world.
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-4">
+          <div class="oppia-donate-page-money-card">
+            <div class="container">
+              <div class="row">
+                <img class="oppia-donate-page-money-icon" [src]="getStaticImageUrl('/donate/oppia-donate-maintenance.svg')">
+              </div>
+              <div class="row oppia-donate-page-money-main-text">Maintenance</div>
+              <hr class="w-25 oppia-donate-page-hr">
+              <div class="row oppia-donate-page-money-sub-text">
+                Donations keep the Oppia platforms and servers running smoothly and reliably.
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-page-card mt-0 mb-0 oppia-donate-page-section-gray-bg">
+    <div class="row justify-content-center">
+      <div class="donate-page-section-heading">Where Our Learners are Saying</div>
+    </div>
+    <div id="donate-page-learner-carousel" class="carousel slide" data-ride="carousel">
+      <ol class="carousel-indicators">
+        <li data-target="#donate-page-learner-carousel" data-slide-to="0" class="active"></li>
+        <li data-target="#donate-page-learner-carousel" data-slide-to="1"></li>
+        <li data-target="#donate-page-learner-carousel" data-slide-to="2"></li>
+      </ol>
+      <div class="carousel-inner">
+        <div class="carousel-item active">
+          <div class="container">
+            <div class="row justify-content-center pt-5">
+              <div class="oppia-donate-page-carousel-img-wrapper">
+                <picture>
+                  <source type="image/webp" [srcset]="getImageSet('/splash/Gaurav', 'webp')">
+                  <source type="image/png" [srcset]="getImageSet('/splash/Gaurav', 'png')">
+                  <img [src]="getStaticImageUrl('/splash/Gaurav', 'png')"
+                       class="rounded-circle oppia-donate-contributor-img"
+                       alt="Logo of Oppia containing photos of contributors">
+                </picture>
+              </div>
+            </div>
+            <div class="row">
+              <div class="carousel-caption text-dark mb-5">
+                <h3 class="text-center">From Khanpur, India</h3>
+                <p class="text-center oppia-donate-student-feedback-text">“I liked learning the lessons on mobile because we get very good questions and if we get something wrong we are told how to correct it and we are not scared when we are doing this I really liked this app.”</p>
```

Comment:
Shouldn't this be translatable string?

Reply:
ditto for others

Reply:
Done. I've also added webp version of JPEG images.The banner one is JPG since it needs to be used as background in CSS, webp might not be supported in some browsers --I've made sure to compress it so that it is of a reasonable size.